commands/peers/deletepeer.go

```diff
@@ -3,29 +3,63 @@ package peercommands
 import (
 	"net/http"
 
+	"github.com/gluster/glusterd2/context"
 	"github.com/gluster/glusterd2/peer"
 	"github.com/gluster/glusterd2/rest"
+	"github.com/gluster/glusterd2/rpc/client"
+
+	log "github.com/Sirupsen/logrus"
+	etcdclient "github.com/coreos/etcd/client"
+	etcdcontext "golang.org/x/net/context"
 
 	"github.com/gorilla/mux"
 )
 
 func deletePeerHandler(w http.ResponseWriter, r *http.Request) {
-	p := mux.Vars(r)
+	peerReq := mux.Vars(r)
```

Comment:
Comments are must when the functions are exposed and this function is certainly not since it starts with a small letter. These comments actually get integrated in go doc where you can view the functions accessible from the package and that's why comments explaining what the function does is really needed.